<ui version="4.0" >
 <class>modCalcGeodCoordDlg</class>
 <widget class="QWidget" name="modCalcGeodCoordDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>466</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="KTabWidget" name="tabWidget" >
     <widget class="QWidget" name="interact" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>596</width>
        <height>410</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Interactive Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox" >
           <property name="title" >
            <string>Select Input Coordinates</string>
           </property>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>8</number>
            </property>
            <item>
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="cartRadio" >
                <property name="text" >
                 <string>Cartesian</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="spheRadio" >
                <property name="text" >
                 <string>Geographic</string>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2" >
           <property name="title" >
            <string>Select Ellipsoid Model</string>
           </property>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>8</number>
            </property>
            <item>
             <widget class="KComboBox" name="ellipsoidBox" />
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>81</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="Compute" >
           <property name="text" >
            <string>Convert</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>51</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="Clear" >
           <property name="text" >
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>91</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_3" >
           <property name="title" >
            <string>Cartesian Coordinates</string>
           </property>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>8</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <widget class="QLabel" name="XCartLabel" >
              <property name="text" >
               <string>X (km):</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="YCartLabel" >
              <property name="text" >
               <string>Y (km):</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="ZCartLabel" >
              <property name="text" >
               <string>Z (km):</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="QLineEdit" name="ZGeoBox" />
            </item>
            <item row="0" column="1" >
             <widget class="QLineEdit" name="XGeoBox" />
            </item>
            <item row="1" column="1" >
             <widget class="QLineEdit" name="YGeoBox" />
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4" >
           <property name="title" >
            <string>Geographic Coordinates</string>
           </property>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>8</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="2" column="0" colspan="2" >
             <widget class="QLabel" name="AltGeoLabel" >
              <property name="text" >
               <string>Elevation (meters):</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2" >
             <widget class="QLineEdit" name="AltGeoBox" >
              <property name="text" >
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="LongGeoLabel" >
              <property name="text" >
               <string>Longitude:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="LatGeoLabel" >
              <property name="text" >
               <string>Latitude:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2" >
             <widget class="dmsBox" name="LongGeoBox" >
              <property name="minimumSize" >
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="focusPolicy" >
               <enum>Qt::StrongFocus</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2" >
             <widget class="dmsBox" name="LatGeoBox" >
              <property name="minimumSize" >
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="focusPolicy" >
               <enum>Qt::StrongFocus</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="batch" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Batch Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>8</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Select Parameters in Input File</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="2" >
           <widget class="QCheckBox" name="YGeoCheckBatch" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Y (km):</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2" >
           <widget class="QCheckBox" name="ZGeoCheckBatch" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Z (km):</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="3" >
           <widget class="QLineEdit" name="ZGeoBoxBatch" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="AltGeoBoxBatch" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QCheckBox" name="AltGeoCheckBatch" >
            <property name="text" >
             <string>Elev. (m):</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QCheckBox" name="XGeoCheckBatch" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>X (km):</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="LongGeoCheckBatch" >
            <property name="text" >
             <string>Longitude:</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="dmsBox" name="LongGeoBoxBatch" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="LatGeoCheckBatch" >
            <property name="text" >
             <string>Latitude:</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QLineEdit" name="XGeoBoxBatch" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="dmsBox" name="LatGeoBoxBatch" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <widget class="QLineEdit" name="YGeoBoxBatch" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6" >
         <property name="title" >
          <string>Select Parameters for Output File</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>8</number>
          </property>
          <item>
           <widget class="QRadioButton" name="AllRadioBatch" >
            <property name="text" >
             <string>All parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="InputRadioBatch" >
            <property name="text" >
             <string>Only parameters in input file</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7" >
         <property name="title" >
          <string>Select Filenames</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Output File:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Input File:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="KUrlRequester" name="InputFileBoxBatch" />
          </item>
          <item row="1" column="1" >
           <widget class="KUrlRequester" name="OutputFileBoxBatch" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>151</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="runButtonBatch" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>131</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>520</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>dmsBox</class>
   <extends>QLineEdit</extends>
   <header>widgets/dmsbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
